libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -861,15 +861,16 @@ - (NSString *)scopeQueryParamString {
 - (void) updateCredentials:(NSDictionary *) params {
 
     // Logging event for token refresh flow.
+    SFUserAccount *userAccount = [[SFUserAccountManager sharedInstance] accountForCredentials:self.credentials];
```

Comment:
@khawkins Moving this code to the coordinator delegate (SFAuthenticationManager). Not sure why it was here in the first place.